### PR TITLE
fix metrics crashers, unit tests

### PR DESCRIPTION
**Pull request by mattfenwick**

### CODE REVIEW

**Comment by mattfenwick** [comment]
Fixes #41

File: image/pkg/metrics/metrics.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func init() {
 	totalImagesAnnotated = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "perceptor",
 		Subsystem: "image_perceiver",
-		Name:      "annotations",
+		Name:      "total_annotations",
 		Help:      "images annotated",
```
Comment (rrati) [inline image/pkg/metrics/metrics.go]: Let's make the help "total images annotated"

File: pod/pkg/metrics/metrics.go [hunk 2]
```diff
@@ -91,7 +91,7 @@ func init() {
 	totalPodsAnnotated = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "perceptor",
 		Subsystem: "pod_perceiver",
-		Name:      "annotations",
+		Name:      "total_annotations",
 		Help:      "pods annotated",
```
Comment (rrati) [inline pod/pkg/metrics/metrics.go]: Let's make the help "total pods annotated"

**Comment by rrati** [comment]
@mattfenwick Also, change the help message on original annotations to be "individual image annotations" and "individual pod annotations"

**Pull request merged**